File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/webhook/JsonMapperConfig.java
package com.github.bumblebee.webhook;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Configuration;

@Configuration
public class JsonMapperConfig {

    @Autowired(required = true)
    public void configureJackson(ObjectMapper mapper) {

        mapper.setPropertyNamingStrategy(new PropertyNamingStrategy.LowerCaseWithUnderscoresStrategy());
    }
}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/webhook/JsonMapperConfig.kt
package com.github.bumblebee.webhook

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.PropertyNamingStrategy
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.annotation.Configuration

@Configuration
open class JsonMapperConfig {

    @Autowired
    fun configureJackson(mapper: ObjectMapper) {
        mapper.propertyNamingStrategy = PropertyNamingStrategy.SnakeCaseStrategy()
    }
}
